.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -190,26 +191,27 @@ jobs:
       run: rustup component add clippy rustfmt
 
     - name: "Rust tools cache"
-      uses: actions/cache@v3
+      uses: actions/cache@v4
       id: rust-tools-cache
       with:
         path: |
           ${{ matrix.cargo_dir }}/bin/sccache${{ matrix.exe_suffix }}
           ${{ matrix.cargo_dir }}/bin/mdbook${{ matrix.exe_suffix }}
           ${{ matrix.cargo_dir }}/bin/mdbook-linkcheck${{ matrix.exe_suffix }}
-        key: ${{ matrix.os }}_sccache-0.4.1_mdbook-0.4.28_mdbook-linkcheck-0.7.7
+        key: ${{ matrix.os }}_sccache-0.7.6_mdbook-0.4.36_mdbook-linkcheck-0.7.7
     - name: "Build Rust tools"
       if: steps.rust-tools-cache.outputs.cache-hit != 'true'
       run: cargo install --locked sccache mdbook mdbook-linkcheck
```

Comment:
sccache has a load of features that we don't need enabled by default, can we install with only the features we require (eg none of the remote stuff)? As this would cause installation to be much faster as it doesn't require a load of web dependencies ...

```
all = [
  "dist-client",
  "redis",
  "s3",
  "memcached",
  "gcs",
  "azure",
  "gha",
  "webdav",
  "oss",
]
```

Reply:
If I read the current PR correctly, you already disabled those features :+1: